wildfly@.service

```diff
@@ -21,6 +21,7 @@ ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 ExecStart=/bin/bash -c 'docker run -P --name ${CONTAINER} --restart=always \
   --env ADMIN_USERNAME=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/wildfly/admin-username?raw) \
   --env ADMIN_PASSWORD=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/wildfly/admin-password?raw) \
+  --env JAVA_OPTS="-Xms1024m -Xmx4096m -XX:MaxPermSize=4096m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true" \
```

Comment:
Yeah, that makes sense. Maybe we'll leave the 64M starting size for the heap for now.

On the PermGen, I was assuming that it was separate from the heap, but you seem to be saying that it takes from the heap? In other words, we thought, "If these both fill up, then WildFly will be using Xmx+PermGen amount of RAM." But are you saying, "It will only use Xmx amount of RAM?"